src/lang/queryAst/getSafeInsertIndex.ts

```diff
@@ -0,0 +1,22 @@
+import { getIdentifiersInProgram } from './getIndentifiersInProgram'
```

Comment:
Adding this comment here so we can do replies and what not.

I found a problem with tags

https://github.com/user-attachments/assets/a8f0dba3-ef88-42f5-9730-9fe80fa8094b

Reply:
Great find thank you! I'll fix that

Reply:
Got 'em


https://github.com/user-attachments/assets/819c2e16-536a-4c34-a151-39432904dcfd

Reply:
Nice.